dev/fattest.simplicity/src/componenttest/rules/repeater/MicroProfileActions.java

```diff
@@ -209,6 +209,9 @@ public class MicroProfileActions extends RepeatActions {
                                                           "mpOpenTracing-3.0",
```

Comment:
I suggest removing this change and the EE compatibility test change and rebase on release or integration and make the changes again with the new way of doing things.